### PR TITLE
Implement cookie handling and program hooks

### PR DESCRIPTION
**Pull request by maklyakov**

### CODE REVIEW

File: app/App.js [hunk 1]
```diff
@@ -51,41 +57,46 @@ export default class App extends Component {
...
-          this.fetchLocation(true);
+          if (res && res.crowd_rating) {
+            const { config } = XcooBee.kit;
+
+            config.cookies.forEach((cookie) => {
+              cookie.checked = res.crowd_rating.find(item => item.cookie_type === cookie.type).value >= 0.8;
+            });
+            this.fetchLocation(true);
+          } else {
+            console.error("Wrong campaign name.");
```
Comment (Bilal-S) [inline app/App.js]: Your error messages especially to console should be prefixed with a constant string like [XcooBee:kit]
```suggestion
            console.error("[XcooBee:kit] Wrong campaign name.");
```

File: app/App.js [hunk 2]
```diff
@@ -168,7 +184,7 @@ export default class App extends Component {
             )
         }
         {
-          (localStorage[tokenKey] || localStorage[xcoobeeCookiesKey]) && (
+          (localStorage[tokenKey] || localStorage[xcoobeeCookiesKey]) && XcooBee.kit.config.testMode && (
```
Comment (Bilal-S) [inline app/App.js]: Is `testMode` a configuration option we need to publish in ReadMe Document.
What does it do?

File: app/utils/index.js [hunk 3]
```diff
@@ -1,18 +1,18 @@
 const defaultConfig = {
-  // campaignReference: "ce84ab7f7b737d",
   // campaignName: "https://lviv.com",
```
Comment (Bilal-S) [inline app/utils/index.js]: name, and reference
These should be removed in final version of code.

Comment (maklyakov) [reply app/utils/index.js]: @Bilal-S Yes, it is for testing now

File: app/App.js (hunk 2 above)
Comment (maklyakov) [reply app/App.js]: @Bilal-S `testMode` adds a refresh button for remove everything for local storage. It's just for easier testing.

**Pull request merged**